vendor/bat-native-ledger/src/bat/ledger/internal/credentials/credentials.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_CREDENTIALS_H_
+#define BRAVELEDGER_CREDENTIALS_H_
+
+#include <map>
+#include <string>
+
+#include "bat/ledger/ledger.h"
+
+namespace braveledger_credentials {
+
+struct CredentialsTrigger {
+  std::string id;
+  ledger::CredsBatchType type;
+  int step;
+  int size;
+};
+
+class Credentials {
+ public:
+  virtual ~Credentials() = default;
+
+  virtual void Start(
+      const CredentialsTrigger& trigger,
+      ledger::ResultCallback callback);
+
+  virtual void BlindedCreds(
+      const ledger::Result result,
+      const std::string& blinded_creds_json,
+      const CredentialsTrigger& trigger,
+      ledger::ResultCallback callback);
+
+  virtual void ClaimCreds(
```

Comment:
nit: do we need `Creds` as the class is called `credentials`?

Reply:
done